internal/blueprint/customizations.go

```diff
@@ -380,3 +380,10 @@ func (c *Customizations) GetOpenSCAP() *OpenSCAPCustomization {
 	}
 	return c.OpenSCAP
 }
+
+func (f *FDOCustomization) HasFDO() *FDOCustomization {
+	if f == nil {
+		return nil
+	}
+	return f
```

Comment:
just `return f != nil` here and make it return a bool (as we're then using `fdo` anyway)

Reply:
done, changes updated in latest commit